Gruntfile.js

```diff
@@ -30,13 +30,22 @@ module.exports = function(grunt) {
     'webpack:functional_tests',
   ]);
 
-  grunt.registerTask('test', [
-    'lint',
-    'flowbin:check',
-    'build-tests',
-    'mochaTest:unit',
-    'mochaTest:functional',
-  ]);
+  grunt.registerTask('test', 'run linting and test suites', function() {
+    var tasks = [
+      'lint',
+      'flowbin:check',
+      'build-tests',
+      'mochaTest:unit',
+      'mochaTest:functional',
+    ];
+
+    if (process.env.APPVEYOR) {
+      tasks = tasks.filter((t) => t !== 'flowbin:check');
```

Comment:
Cool. Could you also file a bug in web-ext (or maybe in Appveyor if that's easy?) so we don't forget to fix this. I guess it's low priority but since flowbin:check seems to work on Windows AFAIK we should investigate.